python/cudf/cudf/utils/utils.py

```diff
@@ -438,3 +439,92 @@ def raise_iteration_error(obj):
         f"Consider using `.to_arrow()`, `.to_pandas()` or `.values_host` "
         f"if you wish to iterate over the values."
     )
+
+
+def arange(start, stop=None, step=1, dtype=None):
```

Comment:
This just doesn't feel like quite the right place for these functions. Any ideas? Maybe static methods on `Column` or something similar?

Reply:
Else similar to `as_column` we can place it in column.py, then we can access it as `cudf.core.column.full` and `cudf.core.column.arange`, I wanted to move them to top like `cudf.full` similar to cupy, but it is odd.

Reply:
I thought about exposing them publicly as well, but until we make `cudf.Column` equivalent to `pandas.Array` including supporting it publicly I don't think we want to do that yet.

Doing it under `cudf.core.column.xxx` sounds good to me!

Reply:
Apologies for the churn in the PR this is causing

Reply:
np, I should have taken care beforehand.